autoload/go/def.vim

```diff
@@ -20,8 +20,8 @@ function! go#def#Jump(...)
 
 	let command = bin_path . " -f=" . expand("%:p") . " -i " . shellescape(arg)
 
-	" get output of godef
-	let out=system(command, join(getbufline(bufnr('%'), 1, '$'), LineEnding()))
+  " get output of godef
+  let out=system(command, join(getbufline(bufnr('%'), 1, '$'), LineEnding()))
```

Comment:
OK, I just checked, in def.vim it use tab as indent, but in my environment I use space. I'll revert it.